### ISSUE TITLE
Handle few deprecations in scipy

### ISSUE THREAD

**Issue by raamana** [loli/medpy#95]
during CI on my [hiwenet](GitHub.com/raamana/hiwenet), I discovered a ton of deprecation warnings from scipy : https://travis-ci.org/github/raamana/hiwenet/jobs/658810855

mostly asking medpy to move from scipy to numpy for functions such as sum, absolute, sqrt, square etc .. see below for a sample, and the above link for a full list.. If you can, can you try fix them? thanks.

```bash
  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:126: 
DeprecationWarning: scipy.absolute is deprecated and will be removed in SciPy 2.0.0, use numpy.absolute instead
367    return scipy.sum(scipy.absolute(h1 - h2))

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:126: 
DeprecationWarning: scipy.sum is deprecated and will be removed in SciPy 2.0.0, use numpy.sum instead
464    return scipy.sum(scipy.absolute(h1 - h2))

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:101: 
DeprecationWarning: scipy.absolute is deprecated and will be removed in SciPy 2.0.0, use numpy.absolute instead
561    mult = scipy.absolute(h1 - h2)

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:103: 
DeprecationWarning: scipy.multiply is deprecated and will be removed in SciPy 2.0.0, use numpy.multiply instead
658    for _ in range(p - 1): dif = scipy.multiply(dif, mult)

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:104: 
DeprecationWarning: scipy.sum is deprecated and will be removed in SciPy 2.0.0, use numpy.sum instead
755    return math.pow(scipy.sum(dif), 1./p)

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:137: 
DeprecationWarning: scipy.absolute is deprecated and will be removed in SciPy 2.0.0, use numpy.absolute instead
852    return math.sqrt(scipy.sum(scipy.square(scipy.absolute(h1 - h2))))

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:137: 
DeprecationWarning: scipy.square is deprecated and will be removed in SciPy 2.0.0, use numpy.square instead
949    return math.sqrt(scipy.sum(scipy.square(scipy.absolute(h1 - h2))))
950
951

  /home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/medpy/metric/histogram.py:676: 
DeprecationWarning: scipy.sqrt is deprecated and will be removed in SciPy 2.0.0, use numpy.lib.scimath.sqrt instead
1519    result = scipy.sum(scipy.sqrt(h1 * h2))
```

**Comment by loli** [comment]
Thanks for telling me about this. I don't have much time currently.
Probably won't change before the next release.

You are welcome to submit a patch, though.

**Comment by StellarStorm** [comment]
This should be fixed in the 0.5.0 release 🎉 
Relevant commit: ee89c059d0445ad97ea005194cbf571294b3b3ca
